php-classes/Slate/CBL/DemonstrationSkill.php

```diff
@@ -22,6 +22,9 @@ class DemonstrationSkill extends \ActiveRecord
             'type' => 'tinyint',
             'unsigned' => true
         ]
+        ,'Demonstrated' => [
```

Comment:
@jmealo so do you want to commit a deletion of this change into this PR or just keep a mental note of it and remove it in a later PR?